action.yml

```
@@ -51,6 +51,10 @@ inputs:
   source-url:
     description: A URL pointing to a `magic-nix-cache` binary. Overrides all other `source-*` options.
     required: false
+  strict-mode:
+    description: Whether to fail when any errors are thrown. Used only to test the Action; do not set this in your own workflows.
+    default: false
+    required: false
```

```suggestion
  _internal-strict-mode:
    description: Whether to fail when any errors are thrown. Used only to test the Action; do not set this in your own workflows.
    default: false
    required: false
```

How about something like this, to make the internalyness more obvious?